### DISCUSSION TITLE
Couldn't install requirements

### DISCUSSION THREAD

**Discussion by rockusho7** [lllyasviel/Fooocus#3598] [General]
Hi!! When I tried to install it, I got the next error:

Installing requirements
Couldn't install requirements.
Command: "I:\Foocus\python_embeded\python.exe" -m pip install -r "requirements_versions.txt" --prefer-binary
Error code: 1

CMD Failed requirements: install -r "requirements_versions.txt"

how can I do to fix it?

**Comment by Yogesh-DevHub** [comment]
Is this the error you encountered?
https://youtu.be/N7ldCUTX4eA?si=zzYJ6cjjtAk0woKh
